libs/platform/table/table.component.scss

```diff
@@ -648,3 +653,7 @@ fdk-dynamic-portal {
 th.fd-table__cell .fd-table__inner {
     font-weight: 700;
 }
+
+.tr-hovered {
```

Comment:
inappropriate naming, it should be something like fd-classNameExample, or it should be styled directly in template